src/column-layout/interfaces.ts

```diff
@@ -50,4 +50,8 @@ export namespace ColumnLayoutProps {
 
 export interface InternalColumnLayoutProps extends ColumnLayoutProps, InternalBaseComponentProps {
   __breakpoint?: ColumnLayoutBreakpoint;
+  /**
+   * Overrides the default wrapper HTML tag.
+   */
+  tagOverride?: 'dl';
```

Comment:
[minor] we typically prefix internal-only properties with `__` (e.g. `__breakpoint` above)so that it's a bit clearer to see at a glance which are public properties and which not